apps-rendering/src/components/MainMedia/MainMediaImage/MainMediaImage.defaults.tsx

```diff
@@ -0,0 +1,101 @@
+import type { SerializedStyles } from '@emotion/react';
+import { css } from '@emotion/react';
+import ImageDetails from '@guardian/common-rendering/src/components/imageDetails';
+import Img from '@guardian/common-rendering/src/components/img';
+import type { Sizes } from '@guardian/common-rendering/src/sizes';
+import type { ArticleFormat } from '@guardian/libs';
+import { ArticleDisplay } from '@guardian/libs';
+import { from, remSpace } from '@guardian/source-foundations';
+import { some } from '@guardian/types';
+import type { Image } from 'image';
+import type { FC } from 'react';
+import { wideContentWidth } from 'styles';
+
+// ----- Setup ----- //
+
+const captionId = 'header-image-caption';
+
+// ----- Subcomponents ----- //
+
+interface CaptionProps {
+	format: ArticleFormat;
+	image: Image;
+}
+
+export const Caption: FC<CaptionProps> = ({ format, image }: CaptionProps) => {
+	switch (format.display) {
+		case ArticleDisplay.Immersive:
+			return null;
+		default:
+			return (
+				<ImageDetails
+					caption={image.nativeCaption}
+					credit={image.credit}
+					supportsDarkMode
+					id={captionId}
+				/>
+			);
+	}
+};
+
+// ----- Component ----- //
+
+const defaultStyles = css`
+	margin: 0 0 ${remSpace[1]} 0;
+	position: relative;
+
+	${from.wide} {
+		width: ${wideContentWidth}px;
+		margin-left: auto;
+		margin-right: auto;
+	}
+`;
+
+const defaultSizes: Sizes = {
+	mediaQueries: [{ breakpoint: 'wide', size: '620px' }],
+	default: '100vw',
+};
+
+export const imgStyles = (
+	width: number,
+	height: number,
+): SerializedStyles => css`
+	display: block;
+	width: 100%;
+	height: calc(100vw * ${height / width});
+
+	${from.wide} {
+		width: ${wideContentWidth}px;
+		height: ${(wideContentWidth * height) / width}px;
+	}
+`;
+
+interface DefaultProps {
+	image: Image;
+	className?: SerializedStyles;
+	format: ArticleFormat;
+}
+
+const DefaultMainMediaImage: FC<DefaultProps> = ({
```

Comment:
You've created this, but I don't see that you've used it anywhere even though the markup returned by your sub components is near enough identical at first glance. Might be something we should talk about